Added main_inf_restart.py and additional files.

Keith-UCT opened the pull request:
Disclaimer: I'm learning how to use python and github.
Edit of main_infinite.py to make it restart-able after exit. Saves a list of all integers that appear in a sequence to main_num.txt and each sequence of 1 or more to main_seq.txt Sequence terminates upon reaching any number already in main_num.txt I'm working to find a way to add the numbers in the sequence after the sequence terminates rather than during to cut down on code length.

Keith-UCT committed:
Added main_inf_restart.py and additional files.

Disclaimer: I'm learning python.
Edit of main_infinite.py to make it restart-able after exit. Saves a list of all integers that appear in a sequence to main_num.txt and each sequence of 1 or more to main_seq.txt Sequence terminates upon reaching any number already in main_num.txt I'm working to find a way to add the numbers in the sequence after the sequence terminates rather than during to cut down on code length.

Abhijeetbyte reviewed (changes requested):
Hi, Keith-UCT

I appreciate your efforts, add a subsection inside **[3]** in the readme file and write briefly about your work. 

Thank you

Keith-UCT commented:
To clarify, will I be adding to subsection [3], or will I add a subsection [4] below it to give detail on main_inf_restart.py?

Keith-UCT committed:
Update README.md

Added subsection [ 4 ] description for main_inf_restart.py and its supporting txt files.

Keith-UCT committed:
Update README.md

Keith-UCT committed:
Update main_num.txt

added description comment

Keith-UCT committed:
Update main_seq.txt

added description comment

Abhijeetbyte reviewed (approved):
Thank you Keith-UCT for your contribution to this repository. I'm glad to merge your pull request
